railties/lib/rails/generators/rails/app/templates/gitignore

```diff
@@ -7,9 +7,6 @@
 # Ignore bundler config.
 /.bundle
 
-# Ignore master key for decrypting credentials and more.
-/config/master.key
```

Comment:
By the way, why does it need the leading slash at all?

Reply:
If it does not contain leading slash, will ignore the files in the subdirectory, so I think that it contains leading slash to avoid it. 
(Since 1851791803e58dfa19dfe1cb686cf5870e655b27)